src/app/applications/applications-menu/applications-menu.component.ts

```diff
@@ -114,4 +119,9 @@ export class CBPApplicationsMenuComponent implements OnInit, OnDestroy {
 
     get toolbarExpanded(): boolean { return this._isToolbarExpanded; }
 
+
+    // TODO find a way to share this
+    stopPropogation($event: Event) {
```

Comment:
Is this supposed to be stopPropogation vs stopPropagation?

Reply:
Typo - it should be `stopPropagation`